src/math/Ray.js

```diff
@@ -276,9 +276,11 @@ THREE.Ray.prototype = {
 
 	}(),
 
-	intersectsSphere: function ( sphere ) {
+	intersectsSphere: function ( sphere, threshold ) {
```

Comment:
I would not change this method. The other methods of this class do not have a tolerance parameter.

If this method is not doing what you want, then don't use it -- inline the code, instead.